pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -57,6 +61,139 @@ var (
+		subsystem,
+		"vertical_scaling_constraints_container_enabled",
+		dpaContainerLabels,
+		"Tracks whether vertical scaling is enabled (1) or disabled (0) for a container from DatadogPodAutoscaler.spec.constraints.containers",
+		commonOpts,
+	)
+
+	// telemetryVerticalScaleConstraintsContainerCPURequestMin tracks minimum CPU request constraint per container
+	telemetryVerticalScaleConstraintsContainerCPURequestMin = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"vertical_scaling_constraints_container_cpu_request_min",
+		dpaContainerLabels,
+		"Tracks minimum CPU request (in millicores) per container from DatadogPodAutoscaler.spec.constraints.containers.requests.minAllowed",
+		commonOpts,
+	)
+
+	// telemetryVerticalScaleConstraintsContainerMemoryRequestMin tracks minimum memory request constraint per container
+	telemetryVerticalScaleConstraintsContainerMemoryRequestMin = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"vertical_scaling_constraints_container_memory_request_min",
+		dpaContainerLabels,
+		"Tracks minimum memory request (in bytes) per container from DatadogPodAutoscaler.spec.constraints.containers.requests.minAllowed",
+		commonOpts,
+	)
+
+	// telemetryVerticalScaleConstraintsContainerCPURequestMax tracks maximum CPU request constraint per container
+	telemetryVerticalScaleConstraintsContainerCPURequestMax = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"vertical_scaling_constraints_container_cpu_request_max",
+		dpaContainerLabels,
+		"Tracks maximum CPU request (in millicores) per container from DatadogPodAutoscaler.spec.constraints.containers.requests.maxAllowed",
+		commonOpts,
+	)
+
+	// telemetryVerticalScaleConstraintsContainerMemoryRequestMax tracks maximum memory request constraint per container
+	telemetryVerticalScaleConstraintsContainerMemoryRequestMax = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"vertical_scaling_constraints_container_memory_request_max",
+		dpaContainerLabels,
+		"Tracks maximum memory request (in bytes) per container from DatadogPodAutoscaler.spec.constraints.containers.requests.maxAllowed",
+		commonOpts,
+	)
+
+	// telemetryStatusHorizontalCurrentReplicas tracks the current replicas value
+	telemetryStatusHorizontalCurrentReplicas = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"status_current_replicas",
+		dpaLabels,
+		"Tracks the current replicas value from DatadogPodAutoscaler.status.currentReplicas",
+		commonOpts,
+	)
+
+	// telemetryStatusHorizontalDesiredReplicas tracks the desired replicas value
+	telemetryStatusHorizontalDesiredReplicas = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"status_desired_replicas",
+		dpaLabels,
+		"Tracks the current replicas value from DatadogPodAutoscaler.status.horizontal.desiredReplicas",
+		commonOpts,
+	)
+
+	// telemetryStatusVerticalDesiredPodCPURequest tracks the sum of CPU requests for all containers
+	telemetryStatusVerticalDesiredPodCPURequest = telemetry.NewGaugeWithOpts(
```

Comment:
It can already be done with the sum of `telemetryVerticalScaleReceivedRecommendationsRequests` and `telemetryVerticalScaleReceivedRecommendationsLimits`